src/Slim/Middleware/Session.php

```diff
@@ -43,6 +43,7 @@ public function __construct($settings = [])
             'httponly'    => false,
             'name'        => 'slim_session',
             'autorefresh' => false,
+            'csrf'        => false
```

Comment:
How do you disable a page from CSRF? I mean, the session should be started but not necessarily *verified*, so it's safe to start the session anyway.

Starting the session in the constructor have same effect as inside `__invoke__`, Slim calls middleware when a request is made.